Maskbook/Service/SchemeService/SchemeService.swift

```diff
@@ -71,6 +84,30 @@ class SchemeService {
         let personaImportItem = PersonaImportItem(type: .mnemonic(mnemonic: mnemonic), name: nickname)
         personaImportHandler.checkExistAndRestore(from: personaImportItem)
     }
+    
+    func personaMnemonicURLEscaped(string: String) -> String {
```

Comment:
Better add some comments for detail of this function.

Reply:
ok